### PR TITLE
add 2019 EC

### PR DESCRIPTION
**Pull request by raynamharris**

### CODE REVIEW

File: pages/governance.md [hunk 1]
```diff
@@ -17,6 +17,18 @@ The Carpentries Executive Council is the highest leadership body of The Carpentr
  
 #### Executive Council Members
 
+**2019**
+
+- Karen Cranston (appointed in 2018)
+- Mesfin Diro Chaka (elected in 2019)
```
Comment (fmichonneau) [inline pages/governance.md:23]: appointed in 2019

File: pages/governance.md [hunk 2]
```diff
@@ -17,6 +17,18 @@ The Carpentries Executive Council is the highest leadership body of The Carpentr
...
+- Kate Hertweck (appointed in 2018)
+- Amy Hodge (elected in 2018)
```
Comment (fmichonneau) [inline pages/governance.md:25]: (re-)elected in 2019

File: pages/governance.md [hunk 3]
```diff
@@ -17,6 +17,18 @@ The Carpentries Executive Council is the highest leadership body of The Carpentr
...
+- Joslynn Lee (elected in 2019)
```
Comment (fmichonneau) [inline pages/governance.md:26]: appointed in 2019

File: pages/governance.md [hunk 4]
```diff
@@ -17,6 +17,18 @@ The Carpentries Executive Council is the highest leadership body of The Carpentr
...
+**2019**
+
+- Karen Cranston (appointed in 2018)
+- Mesfin Diro Chaka (elected in 2019)
+- Kate Hertweck (appointed in 2018)
+- Amy Hodge (elected in 2018)
+- Joslynn Lee (elected in 2019) 
+- Lex Nederbragt (elected in 2018)
+- Raniere Silva (elected for 2018)
+- Juan Steyn (elected in 2019)
```
Comment (fmichonneau) [inline pages/governance.md:29]: appointed in 2019

File: pages/governance.md [hunk 5]
```diff
@@ -17,6 +17,18 @@ The Carpentries Executive Council is the highest leadership body of The Carpentr
...
+- Elizabeth Wickes (elected in 2018)
```
Comment (fmichonneau) [inline pages/governance.md:30]: (re-)elected in 2019

**Review by fmichonneau — CHANGES_REQUESTED** [review]
Edits based on https://carpentries.org/blog/2018/12/ec-election-results/

**Comment by kariljordan** [comment]
I reviewed and merged these changes. Thank you @fmichonneau for the fixes!

**Pull request merged**